docker/README.md

```diff
@@ -1,80 +1,161 @@
-# Using Docker with cve-services
+# Using Docker with CVE Services
 
 ## Security Considerations
 
-The cve-services docker container contains a MongoDB instance that doesn't require authentication. If this container is exposed to the public internet it may allow unauthorised actors to modify data stored in the database.
+>**Warning**
+>
+>DO NOT use the dev configuration on a public network. The dev environment includes credentials to enable rapid development, and the MongoDB service config does not include authentication. This is not secure for public deployment.
 
-## Setup for Local Development
+## Local Development and Testing
 
-Use the following steps to build and run cve-services (Node.js app
-and Mongo) in docker containers.  Note that the current configurations
-are set to run the services in development or staging (testing instance) mode. Note that these commands
-must be run from the `docker/` directory.
+The following builds and runs cve-services (Node.js app and Mongo) in Docker containers. The included example configurations run the services in `development` or `integration` mode.
 
-The docker setup uses an environment defined in `docker/.docker-env.example`
-but `docker-compose` looks for this file at `docker/.docker-env`.  The
-`docker/.docker-env` file is ignored by our `.gitignore`. This is a
-safeguard against developers committing changes to their local docker
-environment.
+>**Note**
+>
+> These commands must be run from the `docker/` directory.
 
-## Setup for Local Penetration Testing
+### Environments
 
-1. Change to the "staging" branch (this branch is used for the public testing instance):
-  `git checkout staging`
+See the `.docker-env.example` and `.docker-env.integration-example` files for examples of required env vars. For development environments, it's important to define the following:
 
-2. Create your environment file if it doesn't already exist:
-  `cp .docker-env.staging-example .docker-env`
+```
+LOCAL_KEY=XXXXXXX-XXXXXXX-XXXXXXX-XXXXXXX
+NODE_ENV=development
+```
+
+When set to a development environment, the `LOCAL_KEY` is the API token key for the secretariat. In integration environments, the API keys will be written to a file named `user-secrets.txt` in the project directory.
+
+### Start Docker Containers
+
+1. Switch to the correct git branch. For penetration testing, use `integration`. For local development, use `dev`.
```

Comment:
'integration' should be 'int' since that is the name of the branch. The file name could be changed to .docker-env.int-example, but could stay as is